db/migrate/20200901014756_create_breeds.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBreeds < ActiveRecord::Migration[5.1]
+  def change
+    create_table :breeds do |t|
+      t.string :name
+      t.string :country
+      t.string :description
+      t.string :images
+      t.boolean :favorite, default: false
```

Comment:
Added this to the migration since we will building out the capability to add a breed to favorites. My first instinct would be to associate Favorites with a user, but this application in its present form has no users or authentication. I may return to amend this later if I realize that Favorite should be its own model and table in the database...